source/common/network/BUILD

```diff
@@ -352,6 +352,23 @@ envoy_cc_library(
     ],
 )
 
+envoy_cc_library(
```

Comment:
Does this only get compiled on win32? Can we make it so?

Reply:
No it is not and it was deliberate. I have no strong opinion here but here is my reasoning behind it:

Con: Increased compilation times on POSIX

Pros:  Cross platform testing

By making it into a library that gets compiled into both platforms we can have the tests running into both platforms. 

The reason I prefer that is because it makes development/refactoring easier for all developers. I want to minimize the probability that a POSIX dev want to make a refactor in this part of the code and is blocked or stuck because they can't run tests for the component.

Reply:
OK fair enough.